spec/features/idv/analytics_spec.rb

```diff
@@ -198,7 +198,7 @@
         flow_path: 'standard', step: 'verify', acuant_sdk_upgrade_ab_test_bucket: :default, skip_hybrid_handoff: nil, analytics_id: 'Doc Auth'
       },
       'IdV: doc auth verify proofing results' => {
-        success: true, errors: {}, flow_path: 'standard', address_edited: false, address_line2_present: false, analytics_id: 'Doc Auth', ssn_is_unique: true, step: 'verify', acuant_sdk_upgrade_ab_test_bucket: :default, skip_hybrid_handoff: nil,
+        ab_tests: nil, acuant_sdk_upgrade_ab_test_bucket: :default, address_edited: false, address_line2_present: false, analytics_id: 'Doc Auth', errors: {}, flow_path: 'standard', irs_reproofing: nil, lexisnexis_instant_verify_workflow_ab_test_bucket: nil, opted_in_to_in_person_proofing: nil, skip_hybrid_handoff: nil, ssn_is_unique: true, step: 'verify', success: true,
```

Comment:
When updating these specs, I sorted the keyword args alphabetically before adding the missing ones